tests/_test_encrypted_state

```diff
@@ -36,6 +36,7 @@ trap "cleanup" EXIT
 
 [ "${SWTPM_INTERFACE}" == cuse ] && source ${TESTDIR}/test_cuse
 source ${TESTDIR}/common
+skip_test_no_tpm12 "${SWTPM_EXE}"
```

Comment:
I think what you did is better. Let me adjust.

Reply:
I updated my branch now with a replacement patch.

Reply:
But you dropped the 2.0 skips? Any of the two versions can be disabled, both are needed.

Reply:
> 
> 
> But you dropped the 2.0 skips? Any of the two versions can be disabled, both are needed.

I only updated my patch ...

Reply:
Hmm, my patch for 2.0 doesn't work: 
common: line 852: /src/swtpm/swtpm: No such file or directory

Reply:
ah I see, I should update based on your changes. ok